react/features/base/testing/components/TestConnectionInfo.js

```diff
@@ -216,6 +228,7 @@ function _mapStateToProps(state) {
     return {
         _conferenceConnectionState: state['features/testing'].connectionState,
         _conferenceJoinedState: conferenceJoined.toString(),
+        _grantModeratorAvailable: typeof APP.conference._room.grantOwner === 'function',
```

Comment:
i still see this logic here, forgot to push?

Reply:
github is being weird, lemme close and re-open :s